### ISSUE TITLE
Funding request form: update text

### ISSUE THREAD

**Issue by OscarSeip** [softwaresaved/lowfat#811]
In the funding request form - section "Is a Purchase Order Required". It currently states in the third paragraph: 
"If the supplier already exists on our system, there should be minimal delay in getting the PO issued. If the supplier is a new supplier, the accounts payable team will contact the supplier for additional information, and **it can take up to 3 months for the PO to be issued.**"

the latter should be changed to "it can take up to 2 months for the PO to be issued".
